helix-term/Cargo.toml

```diff
@@ -67,6 +67,9 @@ serde = { version = "1.0", features = ["derive"] }
 grep-regex = "0.1.10"
 grep-searcher = "0.1.10"
 
+# case conversion
+heck = "0.4.0"
```

Comment:
Pulls in another dependency :/

Reply:
`heck` has only one dependency : [unicode-segmentation = { version = "1.2.0", optional = true }](https://github.com/withoutboats/heck/blob/dbcfc7b8db8e532d1fad44518cf73e88d5212161/Cargo.toml#L20)

Reply:
Yeah but heck is a new dependency that's pulled in for just a couple commands.